src/components/HallOfFameButton.js

```diff
@@ -0,0 +1,20 @@
+const ManuButton = (firstLetterButtonInnerText, restOfTextButtonInnerText) => {
+
+    const button = document.createElement('button');
+    button.classList.add('displayButton');
+
+    const firstLetter = document.createElement('span');
+    firstLetter.classList.add('hallOfFameButton__greenBlock');
```

Comment:
Jeśli to ogólny przycisk do wielokrotnego wykorzystania to raczej
```suggestion
    firstLetter.classList.add('menuButton__greenBlock');
```